website/static/js/addonPermissions.js

```diff
@@ -38,7 +38,7 @@ var AddonPermissionsTable = {
                             },
 
                             error: function () {
-                                $osf.growl('An error occurred, the project has not been deauthorized. ',
+                                $osf.growl('An error occurred, the Account is still connected to the project. ', 
```

Comment:
Same here; don't capitalize "account". Also, it might be nice to add the provider name here.